src/SIAN.jl

```diff
@@ -312,7 +312,7 @@ function identifiability_ode(ode::ModelingToolkit.ODESystem, params_to_assess=[]
       throw(error("Measured quantities (output functions) were not provided and no outputs were found."))
     end
   end
-  ode_prep, input_syms, gens_ = PreprocessODE(ode, measured_quantities)
+  ode_prep, input_syms, gens_ = preprocess_ode(ode, measured_quantities)
```

Comment:
It looks like the new preprocess_ode now only returns two outputs, instead of three. It combines the last two outputs into a single dictionary